Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM_test.py

```diff
@@ -318,6 +319,8 @@ def test_deduplication(self, mocker, client):
         assert total_events_count == 247
         for event_not_in_list in events_not_in_list:
             assert event_not_in_list not in events
+        ctx = {"offset": offset, "hashed_events_from_previous_run": list(hashed)}
+        assert isinstance(json.dumps(ctx), str)
 
 
```

Comment:
lets add a test where we call fetch 3 times with different mocked events, one where an event should be filtered out, and see we're storing the context properly

Reply:
I have tests which ensure the dedup is correct, the obtained results are json serializable, and that the returned list of hashes includes the events that were deduped